mappings/net/minecraft/world/gen/feature/Feature.mapping

```diff
@@ -32,3 +32,10 @@ CLASS net/minecraft/class_3031 net/minecraft/world/gen/feature/Feature
 		ARG 0 feature
 	METHOD method_28629 (Lnet/minecraft/class_3037;)Lnet/minecraft/class_2975;
 		ARG 1 config
+	METHOD method_33981 isAir (Ljava/util/function/Function;Lnet/minecraft/class_2338;)Z
```

Comment:
`isAir` could be misleading since this method doesn't check if the block at `pos` is air. It returns true if any of the adjacent blocks is air.

Reply:
So like `isExposedToAir`?

Reply:
`isExposedToAir` is better because that's the wording that Mojang used in their article, will do that

Reply:
```suggestion
	METHOD method_33981 isExposedToAir (Ljava/util/function/Function;Lnet/minecraft/class_2338;)Z
```